--- a/.vite/build/worker.js
+++ b/.vite/build/worker.js
@@ -93350,6 +93350,7 @@
   let n = qz(e.appVersion),
     r = e.buildFlavor !== `prod`;
   (yT({
+    enabled: !1,
     dsn: e.dsn,
     environment: e.buildFlavor,
     release: gU(n.version),
--- a/.vite/build/window-all-closed-BwlaNiSa.js
+++ b/.vite/build/window-all-closed-BwlaNiSa.js
@@ -41122,6 +41122,7 @@
   t && (s = +!!a);
   let c = pe(e.version);
   return (
     qz({
+      enabled: !1,
       dsn: uV.dsn,
       environment: uV.buildFlavor,
       release: c,
