--- a/webview/assets/app-initial-BTphDPeq.js
+++ b/webview/assets/app-initial-BTphDPeq.js
@@ -382950,7 +382950,7 @@
     he = (0, zW.useRef)(ee.get()),
     ge = (0, zW.useRef)(new Set()),
     _e = {
-      "--spacing-token-safe-header-left": `${w.left / C + 6}px`,
+      "--spacing-token-safe-header-left": `0px`,
       "--spacing-token-safe-header-right": `${w.right / C}px`,
       width: `calc(100vw / var(--codex-window-zoom))`,
       height: `calc(100vh / var(--codex-window-zoom))`,
